packages/serverless-helpers/README.md

```diff
@@ -0,0 +1,17 @@
+# @swarmion/serverless-helpers
+
+A set of helpers for Serverless Swarmion applications
```

Comment:
Not only for `swarmion` applications, most of the helpers can be used without :wink: